nornir_nautobot/constants.py

```diff
@@ -194,6 +194,18 @@
         error_message="Discovered `% Permission denied for the role` in the output",
         recommendation="Ensure that the actual command that is ran, is allowed for the user making the connection. As an example, if `show run` is allowed, but `show running-config` is not, would need to address that.",
     ),
+    "E1031": ErrorCode(
+        troubleshooting="Ensure that the output file for the command `{command}` has been created.",
+        description="While using offline command outputs through Git, the output file for the command `{command}` could not be retrieved.",
+        error_message="The command output file for `{command}` could not be retrieved.",
+        recommendation="Ensure that the output file for the command `{command}` has been created.",
+    ),
+    "E1032": ErrorCode(
+        troubleshooting="Ensure that the output file for the command `{command}` has been created.",
+        description="While using offline command outputs through Git, the output file for the command `{command}` was not found.",
+        error_message="The command output file for `{command}` could not be found.",
+        recommendation="Ensure that the output file for the command `{command}` has been created.",
+    ),
```

Comment:
Mind updating these to be a bit more distinct?  Appears that 1031 is an issue with loading the file? and 1032 is the file does not exist?  If that is the case can we make the troubleshooting better.